libraries/base/Data/Bifunctor.hs

```diff
@@ -59,22 +67,49 @@ class Bifunctor p where
     -- | Map over both arguments at the same time.
     --
     -- @'bimap' f g ≡ 'first' f '.' 'second' g@
+    -- 
+    --==== Examples:
+    -- >>> bimap toUpper (+1) ('j', 3)
+    -- ('J',4)
+    -- 
+    -- >>> bimap toUpper (+1) (Left 'j')
+    -- Left 'J'
+    -- 
+    -- >>> bimap toUpper (+1) (Right 3)
+    -- Right 4
     bimap :: (a -> b) -> (c -> d) -> p a c -> p b d
     bimap f g = first f . second g
 
+
     -- | Map covariantly over the first argument.
     --
     -- @'first' f ≡ 'bimap' f 'id'@
+    --
+    --==== Examples:
+    -- >>> first toUpper ('j', 3)
+    -- ('J',3)
+    -- 
+    -- >>> first toUpper (Left 'j')
+    -- Left 'J'
     first :: (a -> b) -> p a c -> p b c
     first f = bimap f id
 
+
     -- | Map covariantly over the second argument.
     --
     -- @'second' ≡ 'bimap' 'id'@
+    --
+    --==== Examples:
```

Comment:
Do you know whether a space is required here? I'm a bit worried that haddock will get confused here.

Reply:
I also do not know whether it's required, but I've fixed my PR to match the other examples of it I could find in `base` (e.g., in Data.List). Sorry for confusion.